src/GraphQLMiddlewareDelegatorFactory.php

```diff
@@ -33,4 +33,17 @@ public function __invoke(
 
         return $pipeline;
     }
+
+    /**
+     * @param ServiceLocatorInterface $serviceLocator the service locator which requested the service
+     * @param string                  $name           the normalized service name
+     * @param string                  $requestedName  the requested service name
+     * @param callable                $callback       the callback that is responsible for creating the service
+     *
+     * @return MiddlewarePipe
+     */
+    public function createDelegatorWithName(ServiceLocatorInterface $serviceLocator, $name, $requestedName, $callback)
```

Comment:
we don't need this anymore. Must implement `Zend\ServiceManager\Factory\DelegatorFactoryInterface`